config/locales/idv/en.yml

```diff
@@ -5,24 +5,28 @@ en:
       activate_by_mail: Activate by mail
       activate_by_phone: Activate by phone
       cancel: Cancel and return to your profile
-      continue: Continue identity verification
+      continue: Continue verifying your identity
       help: Continue to Help Center
       mail:
         resend: Send another letter
         send: Send a letter
       return_to_account: Return to account
       return_to_sp: Return to %{sp}
+      return_to_verify: "‹ Go back to seeing all options for proving my identity"
       send_confirmation_code: Send confirmation code
     cancel:
       modal_header: Are you sure you want to cancel?
       warning_header: If you cancel now
+      warning_qualifier: may not
+      warning_point: Login.gov won't be able to verify your identity and you %{warning_qualifier}
+        be able to access your information at %{sp_name}.
+      warning_point_no_sp: Login.gov won't be able to verify your identity and you
+        %{warning_qualifier} be able to access your information.
       warning_points:
-      - We won’t be able to verify your identity
-      - We won't keep a record of your name, address, birth date, or Social Security
-        number
-      - You won't be able to securely access your information using login.gov
-      - You’ll still have a login.gov account for your email address
-      - You can manage that account on your profile page
+      - Login.gov won't keep a record of your name, address, birth date, or Social
+        Security number.
+      - You’ll still have a login.gov account. You can manage or delete your account
```

Comment:
Users cannot delete their account from their profile. Also, didn't we rename the term  "profile" to "account"?

Reply:
The content for this modal should be:

- Login.gov won’t be able to verify your identity, and you may not be able to access your information at **AgencyName.**
- Login.gov won't keep your name, address, date of birth or Social Security number.
- You'll still have a basic login.gov account.